README.md

```diff
@@ -16,3 +16,21 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+-----
+
+To verify the claim, I could create an algorithm that generates a random list of two elements, a random list of three elements, etc. up to a random list of 1000 elements. The algorithm would give X and a general sorting algorithm the same of each list and keep track of how long it takes the sorting algorithms to sort each list. It would do this by starting two timers, one for each algorithm, when it gives both sorting algorithms the 2 element list and would stop an algorithm’s timer when it sorted all of the lists. Once both algorithms sorted all lists, the times would be compared. If the general sorting algorithm is slower by at least log(n), that means the claim is true since X would be asymptotically faster.
+
+To tell if the general sorting algorithm is slower by at least log(n), I could create a graph, where the x-axis represents the input size and the y-axis represents the time it takes for the algorithm to sort the list.  I could then compare the lines of the general sorting algorithm with X, and if the line of the general sorting algorithm grows more linearly than X, which would be growing logarithmically if it is asymptotically faster, then that means the general sorting algorithm is slower by at least log n.
```

Comment:
And how would this tell you whether the claim is correct? It's about the performance of the algorithm itself, not how it compares to other algorithms.